docusaurus/docs/reference/config-types/host.v1.md

```diff
@@ -43,6 +43,8 @@ properties to indicate what should be forwarded:
 * `allowedProtocols` - the list of allowed protocols. Valid values include `tcp` and `udp`
 * `forwardAddress` - flag indicating that the target address of the intercepted connection is to be
   used. Can only be set to true.
+* `forwardAddressTranslations` - the list of translations that specify which address to connect
+  to for a given target address.
```

Comment:
I feel like a link here and an anchor below to a header like "Foward Addresss Translation Example" would be useful. as is it feels a bit too 'flow-y'